src/libs/Environment/betaChecker/index.android.js

```diff
@@ -11,11 +11,16 @@ function isBetaBuild() {
         fetch(CONST.PLAY_STORE_URL)
             .then(res => res.text())
             .then((text) => {
-                const productionVersionMatch = text.match(/<span[^>]+class="htlgb"[^>]*>([-\d.]+)<\/span>/);
+                const productionVersionMatch = text.match(/\[\[\["\d+\.\d+\.\d+/);
```

Comment:
Is this called `productVersionMatch` because we are using the `text.match` method?
If so I think we should rename it. The word `Match` is confusing here because it implies comparison

Reply:
what about changing it to `productionVersionSearch`, the inner me can't find a good variable name 😂

Reply:
Yea, it is kinda tricky, heh. This is fine